HOWTO_android.md

```diff
@@ -20,23 +20,23 @@ Copyright &copy; 2023 LunarG, Inc.
 
 ## Purpose
 
-The purpose of this doc is to show several examples in using the GFXReconstruct
-tools.
-These examples are intended to help work through issues you may discover while
-attempting to perform the same process on your own application.
+This document shows several examples of how to use the GFXReconstruct tools.
+These examples will help work through issues you may discover while attempting
+to perform the same process on your own application.
 
 **NOTE:**
 Many of these examples are given based on using a Linux system as the build
-and/or host, but should be fairly easy to replicate behavior on a Windows
-system with very little changes.
+and/or host, but you should be able to replicate the behavior on a Windows
+system with very few changes.
 
 ## Building Vulkan Samples with the Capture Layer
 
-Let's look at adding the GFXReconstruct layer as a requirement of the 
+Let's look at adding the GFXReconstruct layer as a requirement of the
 [Khronos Vulkan Samples](https://github.com/KhronosGroup/Vulkan-Samples)
 project.
-These steps assume that you have all the dependencies necessary to build that
-project, so refer to that repo to ensure those are met before continuing.
+Completing the following steps will ensure that you have all the dependencies
```

Comment:
I think your original thought was correct - the following steps all assume you have met all the dependencies to build Vulkan-Samples.

Reply:
I'll revert it

Reply:
I cleaned it up.  Not a direct revert, but near enough.